assets/sass/cds/_components.scss

```diff
@@ -154,6 +154,17 @@
   a:link {
     color: #245269;
   }
+	background: #d7faff;
+	border-left: 4px solid #269abc;
+	padding: 2.2rem 2.5rem;
```

Comment:
The info-alert also shows up on the spa research page, does that stying still look ok?